assets/src/components/activities/adaptive/AdaptiveAuthoring.tsx

```diff
@@ -322,6 +332,11 @@ const Adaptive = (
               <i className="las la-edit"></i>
             </button>
           )}
+          {selectedPart && selectedPart.capabilities.move && (
```

Comment:
the "move" capability is supposed to mean whether it could be dragged, I don't recall if we have a "copy" capability, but maybe we should have one for that